src/bindings/python/src/openvino/frontend/pytorch/utils.py

```diff
@@ -162,6 +163,22 @@ def forward(self, {input_sign}):
 """
 
 
+def build_wrapper(template, model):
+    """
+    Builds a wrapper around the given model using the provided template.
+    """
+    result = {}
+    try:
+        exec(template, result)
+
+        wrapped_model = result["ModelWrapper"](model)
+        wrapped_model.eval()
+    # if wrapping failed, it is better to return original model for avoid user confusion regarding error message
+    except Exception:
+        wrapped_model = model
```

Comment:
I think we should report a warning about exception in verbose mode for debuggability

Reply:
Agree